src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -224,6 +225,15 @@ internal ChunkInheritanceUtility ChunkInheritanceUtility
             }
         }
 
+        public IReadOnlyList<ChunkTreeResult> GetInheritedChunkTreeResults(string sourceFileName)
```

Comment:
Nope, at least not today. It's consumed by tooling who knows of/constructs `MvcRazorHost`